### PR TITLE
Code review request

### PR DESCRIPTION
**Pull request by VanessaCalimag**
Please check formatting, naming and logic (or anything else you want checked) @MitchellK. Thank you!

### CODE REVIEW

File: prac_05/state_names.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# Reformat this file so the dictionary code follows PEP 8 convention
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
```
Comment (MitchellK) [inline prac_05/state_names.py:8]: note that South Australia is missing! :)

File: prac_05/state_names.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
```
Comment (MitchellK) [inline prac_05/state_names.py:10]: white spacing comment should be one line lower.

File: prac_05/word.occurrences.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+"""This program will count the occurrences of words in a string."""
+
+text = {}
+string_of_words = input('Text: ')
+# this is a collection of words of nice words this is a fun thing it is
```
Comment (MitchellK) [inline prac_05/word.occurrences.py:5]: comment can be removed, or places above this section

**Comment by MitchellK** [comment]
Your code is so much better than mine haha! awesome work!

File: prac_05/word.occurrences.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+"""This program will count the occurrences of words in a string."""
+
+text = {}
```
Comment (lindsaymarkward) [inline prac_05/word.occurrences.py:3]: I suggest 99.9% of people seeing a variable called `text` would think it's a `string`.

**Comment by lindsaymarkward** [comment]
Where do you answer:
> Do this next part on paper first (then in PyCharm):
> Write a loop that prints all of the states and names neatly lined up with string formatting

**Pull request merged**